dietpi/dietpi-software

```diff
@@ -11103,6 +11109,34 @@ _EOF_
 			modprobe binfmt_misc 2> /dev/null && G_EXEC systemctl restart systemd-binfmt
 		fi
 
+		if To_Install 206 # Moonlight
+		then
+			## Install script adapted from
+			## - https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-embedded/setup.deb.sh
+
+			# APT deps
+			aDEPS=('apt-transport-https' 'ca-certificates' 'debian-keyring' 'debian-archive-keyring' 'gnupg')
+
+			# Install paths
+			local gpg_keyring_path="/usr/share/keyrings/moonlight-game-streaming-moonlight-embedded-archive-keyring.gpg"
+			local repo_path="/etc/apt/sources.list.d/moonlight-game-streaming-moonlight-embedded.list"
+
+			# APT key
+			local base_url="https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-embedded"
+			local gpg_url="$base_url/gpg.5AEE46706CF0453E.key"
+			G_CHECK_URL "$gpg_url"
+			G_EXEC eval "curl -1sLf '$gpg_url' | gpg --dearmor > '$gpg_keyring_path'"
+
+			# APT list
+			local repo_url="$base_url/deb/raspbian"
+			G_EXEC eval "echo 'deb [signed-by=$gpg_keyring_path] $repo_url $G_DISTRO_NAME main' > $repo_path"
+			G_EXEC eval "echo 'deb-src [signed-by=$gpg_keyring_path] $repo_url $G_DISTRO_NAME main' >> $repo_path"
```

Comment:
Let's align the code with other 3rd party repo installs:
```suggestion
			# Detect distro
			local distro='debian'
			(( $G_HW_MODEL < 10 )) && (( $G_RASPBIAN )) && distro='raspbian'

			# APT key
			local url='https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-embedded/gpg.5AEE46706CF0453E.key'
			G_CHECK_URL "$url"
			G_EXEC eval "curl -sSfL '$url' | gpg --dearmor -o /etc/apt/trusted.gpg.d/dietpi-moonlight.gpg --yes"

			# APT list
			G_EXEC eval "echo 'deb https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-embedded/$distro/ $G_DISTRO_NAME stable' > /etc/apt/sources.list.d/dietpi-moonligh.list"
```
- I'm pretty sure there is a "debian" repo path if Debian is used instead of Raspbian, which enables ARMv8 and x86_64 support.
- Using `signed-by` instead of `trusted.gpg.d` is actually the modern/better way, but I'd like to keep things aligned for now and migrate all 3rd party repos together at some point.
- Skip source package list `deb-src`, not needed for installing binary packages. If really someone wants to compile it manually, its trivial to add it manually.

Reply:
Makes sense!